packages/twenty-server/src/modules/contact-creation-manager/utils/filter-out-contacts-from-company-or-workspace.util.ts

```diff
@@ -21,9 +21,14 @@ export function filterOutSelfAndContactsFromCompanyOrWorkspace(
     new Map<string, boolean>(),
   );
 
+  // TODO : find a better way to enable organization1.onmicrosoft.com to add user from organization2.onmicrosoft.com
+  const isDifferentDomain = (contact: Contact, selfDomainName: string) =>
+    getDomainNameFromHandle(contact.handle) !== selfDomainName ||
+    selfDomainName === 'onmicrosoft.com';
```

Comment:
in this case shouldn't we check the subdomain from onmicrosoft.com?
`|| ( selfDomainName === 'onmicrosoft.com' && subdomain1 <> subdomain2)`

Reply:
new idea implemented. let me know what you think